app/Traits/AccessControlAPI.php

```diff
@@ -190,9 +190,12 @@ public function allowedByBulkAccessControl($fileset)
 
     public function allowedForDownload($fileset)
     {
-        // If the API key belongs to bible.is but the user is not currently logged in,
+        // If the API key belongs to bible.is DBP will utilize the bible.is access group list else,
         // the system will utilize the generic access group list instead.
-        if ($this->doesApiKeyBelongToBibleis($this->key) && isUserLoggedIn()) {
+
+        // Note that in the future, this constraint should be updated to account for
+        // whether the user is logged in or not.
+        if ($this->doesApiKeyBelongToBibleis($this->key)) {
```

Comment:
could you leave this line in, commented out, for ease of implementation when we do it?
if ($this->doesApiKeyBelongToBibleis($this->key) && isUserLoggedIn()) {

Reply:
@bradflood sure, I have added the commented line.